_posts/2022-03-04-february-rubygems-updates.md

```diff
@@ -0,0 +1,36 @@
+---
+title: February 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Central](http://rubytogether.org/), we publish a recap of the work that we've done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in February.
+
+## RubyGems News
+
+This month in RubyGems, we released new versions for RubyGems [3.3.7](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#337--2022-02-09), [3.3.8](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#338--2022-02-23) and Bundler [2.3.7](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#237-february-9-2022), [2.3.8](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#238-february-23-2022).
+
+The following is a non-exhaustive list of the improvements included in the above releases (see changelogs for more further information):
+
+- resolved some long-standing issues with our CI workflow and worked on some long-standing configuration issues - [#5324](https://github.com/rubygems/rubygems/pull/5324).
+- fixed an issue with a corrupt lockfile when you run `bundle check` and have to re-resolve locally - [#5344](https://github.com/rubygems/rubygems/pull/5344).
+- fixed typo in multiple gemfiles warning - [#5342](https://github.com/rubygems/rubygems/pull/5342).
+- added clarification for `bundle-config` `"with"` option  - [#5346](https://github.com/rubygems/rubygems/pull/5346).
+- the `BUNDLE_WITH` and `BUNDLE_WITHOUT` environment variables shouldn't be persistent locally - [#5335](https://github.com/rubygems/rubygems/pull/5335).
```

Comment:
I would tweak this item to 

```suggestion
- Fix issue with `BUNDLE_WITH` and `BUNDLE_WITHOUT` environment variables being silently persisted locally - [#5335](https://github.com/rubygems/rubygems/pull/5335).
```

And we should also add an important change made this month:

```
- `bundle config` now saves configuration locally by default when run inside an application context - [#4512](https://github.com/rubygems/rubygems/pull/4152).
```

Other than that looks good!

Reply:
Got it! Thanks for the suggestion. Will add this change.